cypress/page/shipping-step.page.ts

```diff
@@ -4,7 +4,8 @@ class ShippingStepPage {
 
     constructor() {
         this.termsAndConditionsCheckbox = "#cgv";
-        this.continueCheckoutBtn = ".cart_navigation > .button > span";
+        this.continueCheckoutBtn = ".cart_navigation > .button[name='processCarrier']";
```

Comment:
this can be simplified as `".cart_navigation button[name='processCarrier']"` so we don specify that button has to be a direct child of `.cart_navigation`, instead of that, we say that button is within the `.cart_navigation`, so is less coupled to the HTML structure